### PR TITLE
[CIAC-5684] Deprecate old py2 content entities

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5684

## Description
The following content entities have a v2 and are in py2, they are deprecated in this PR:

**JoeSecurity Pack:**
Joe Security

**Sandblast Pack:**
SbDownload
SbQuery
SbQuota
SbUpload

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.yml [hunk 1]
```diff
@@ -29,12 +29,12 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: Sandbox Cloud
-display: Joe Security
+description: Deprecated. No available replacement.
```
Comment (ShacharKidor) [inline Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.yml:32]: Isn't Joe Security v2 the available replacement for that integration?

File: Packs/JoeSecurity/ReleaseNotes/1_1_7.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Joe Security (Deprecated)
+
+- Deprecated. Use Joe Security v2 instead.
```
Comment (ShacharKidor) [inline Packs/JoeSecurity/ReleaseNotes/1_1_7.md:6]:
```suggestion
- Deprecated. Use **Joe Security v2** instead.
```

File: Packs/Sandblast/ReleaseNotes/1_0_9.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+#### Scripts
+
```
Comment (ShacharKidor) [inline Packs/Sandblast/ReleaseNotes/1_0_9.md:3]: Please use RN entity styling here as well  - [Entity Styiling](https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling).

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Nice :) 

@samuelFain - I've added a few little comments please review. 
Also - I think it is worth adding a word about the deprecation to the README files of each deprecated item.

File: Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.yml (hunk 1 above)
Comment (samuelFain) [reply Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.yml:32]: You are correct, fixed it 👍

File: Packs/JoeSecurity/ReleaseNotes/1_1_7.md (hunk 2 above)
Comment (samuelFain) [reply Packs/JoeSecurity/ReleaseNotes/1_1_7.md:6]: Done.

File: Packs/Sandblast/ReleaseNotes/1_0_9.md (hunk 3 above)
Comment (samuelFain) [reply Packs/Sandblast/ReleaseNotes/1_0_9.md:3]: Done.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**